CHANGELOG.md

```diff
@@ -1,76 +1,81 @@
 # Changelog
 
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html). If you introduce breaking changes, please group them together in the "Changed" section using the **BREAKING:** prefix.
+
 ## [v0.1.1] - 2022-11-24
 
-### Breaking Change
+### Changed
+
+- **BREAKING:** Updated extension to require "Authorization code" grant. Your client ID need to be registered with "authorization_code" and "refresh_token" grant type now
 
-- "Authorization code" grant is applied now. Password and token file approach has been removed. Your client ID need to be registered with "authorization_code" and "refresh_token" grant type now.
+### Added
 
-### Feature
+- Added login with SASLogon with PKCE
+- Added support to refresh access token with refresh token
+- Added built-in client ID for Viya4 2022.11 and later
 
-- Login with SASLogon with PKCE
-- Refresh your access token with refresh token
-- Built-in client ID for Viya4 2022.11 and later
+### Removed
+
+- Removed password and token file login approach
 
 ## [v0.1.0] - 2022-10-08
 
-### Breaking Change
+### Changed
 
-- Command ID `SAS.session.run` and `SAS.session.runSelected` changed to `SAS.run` and `SAS.runSelected`
-- Settings `SAS.session.host`, `SAS.session.clientId`, etc. migrated to `SAS.connectionProfiles`
+- **BREAKING:** Changed Command ID `SAS.session.run` and `SAS.session.runSelected` changed to `SAS.run` and `SAS.runSelected`
```

Comment:
This syntax looks weird: Changed Foo changed to Foo

Reply:
Good catch